Tools/TestWebKitAPI/Configurations/DebugRelease.xcconfig

```diff
@@ -27,6 +27,9 @@
 CODE_SIGN_IDENTITY[sdk=embedded] = $(CODE_SIGN_IDENTITY_EMBEDDED_$(USE_INTERNAL_SDK));
 CODE_SIGN_IDENTITY_EMBEDDED_YES = -;
 
+CODE_SIGN_IDENTITY[sdk=macosx*] = $(CODE_SIGN_IDENTITY_$(SDK_VARIANT)_$(USE_INTERNAL_SDK));
+CODE_SIGN_IDENTITY_iosmac_YES = -;
```

Comment:
I cannot pinpoint any specific problem, but I don't think that "like we do on embedded platforms" is a good enough explanation of why this is workable. Code signing checks for macCatalyst match those of macOS, not iOS or iOS simulator.

I have to admit that I'm somewhat confused about what precisely we are trying to achieve here (not just to fix the open source build, but in other cases). We should discuss that elsewhere.